CityDBMigrations/Migrations/20230223081013_AddCategoryTable.cs

```diff
@@ -15,7 +15,9 @@ public override void Up()
                     noOfSubcategories int DEFAULT 0
                 );
                 INSERT INTO categories (name, noOfSubcategories) values ('News', 8);
-                INSERT INTO categories (name) values ('Road works / Traffic'), ('Events'), ('Clubs'), ('Regional Products'), ('Offer / Search'), ('New citizen info'), ('Defect report'), ('Lost and Found'), ('Company portraits'), ('Carpooling / Public transport'), ('Offers');";
+                INSERT INTO categories (id, name) values (25, 'Polls');
+                INSERT INTO categories (name) values ('Road works / Traffic'), ('Events'), ('Clubs'), ('Regional Products'), ('Offer / Search'), ('New citizen info'), ('Defect report'), ('Lost and Found'), ('Company portraits'), ('Carpooling / Public transport'), ('Offers');
+                ";
```

Comment:
Do not add like this this migration won't run for the cities already created. create a new file and add the category in it.